cfgov/regulations3k/jinja2/regulations3k/regulations-listing.html

```diff
@@ -26,6 +26,18 @@ <h4>{{ value.heading or 'Table of contents' }}</h4>
             <a href="{{ regulation.url }}">
                 <span class="title-num">{{ regulation.title }}</span>
             </a>
+            {% if regulation.regulation.future_versions %}
+            <ul>
+            {% for version in regulation.regulation.future_versions %}
+                <li>New amendment effective 
```

Comment:
You know, I wonder @niqjohnson if this should read "New amendments effective...".

Reply:
Yeah, I like "amendments" instead of the singular. Making that change sounds good to me.